cmd/zqd/listen/command.go

```diff
@@ -258,6 +265,24 @@ func (c *Command) initSuricata() error {
 	return nil
 }
 
+func (c *Command) initWorkers() error {
+	// This is for local testing only, at this point.
+	// Workers will be available through a K8s service in a prod deployment.
+	if c.workers != "" {
+		workerArr := strings.Split(c.workers, ",")
```

Comment:
This is a slice, not an array, so workerArr isn't the best name. I'd just call it workers.